gcp_variant_transforms/beam_io/vcf_header_io.py

```diff
@@ -161,17 +168,18 @@ def __init__(
       compression_type=CompressionTypes.AUTO,
       validate=True,
       **kwargs):
+    # type: (str, str, bool, **str) -> None
```

Comment:
BTW, is **str correct? I thought for variable-length arguments, you just need to define the type of the values, e.g., `str` in this case. [Here](https://github.com/googlegenomics/gcp-variant-transforms/blob/5512e444267d67b826f5ab323904bcff8ea56ef3/gcp_variant_transforms/libs/annotation/vep/vep_runner.py#L196) is one example but please let me know if I have done it wrong.

Reply:
Yes, I think it works for python 3. But for python 2, there is a warning in IntelliJ saying it is expecting a dictionary.
The suggested syntax for Python 2.7, quoted from [PEP 484](https://www.python.org/dev/peps/pep-0484/):
``When using the short form, for *args and **kwds, put 1 or 2 stars in front of the corresponding type annotation. (As with Python 3 annotations, the annotation here denotes the type of the individual argument values, not of the tuple/dict that you receive as the special argument value args or kwds.)``